Lib/gettext.py

```diff
@@ -296,6 +297,7 @@ def ngettext(self, msgid1, msgid2, n):
             return msgid2
 
     def lngettext(self, msgid1, msgid2, n):
+        import locale
```

Comment:
Can you move the locale import after the warn() call? Here in the following function.

Reply:
ok, done.